src/styles/common/_buttons.scss

```diff
@@ -80,6 +80,10 @@
         box-shadow: none;
         opacity: .4;
     }
+
+    & + & {
+        margin-left: 5px;
```

Comment:
It seems like this should be `10px`, since the left/right margins were additive.